openvalidation-generation/src/main/resources/javascript/framework.hbs

```diff
@@ -145,4 +145,12 @@ var HUMLFramework = function() {
     this.NOT_EXISTS = function(operand) {
         return this.EMPTY(operand);
     }
+
+    this.SUM_OF = function(operand){
+        var value = 0;
+        for (let i=0; i<operand.length; i++) {
+            value += operand[i];
+          }
+        return value;
+    }
 }
```

Comment:
Will you implement `this.FIRST` and `this.LAST` in this PR as well?

Reply:
Yes. Maybe it's not very visible, but i will even out the functionality of the three base generated Frameworks. Adding it to the description.

Reply:
I have to confess though, I have just fiddled with the java framework in my Python PR.
I refactored the `getArrayOf` and `where` methods to `GET_ARRAY_OF` and `WHERE`, to be more consistent in the naming of cross-language HUMLFramework functions.
I hope this won't bring up nasty conflicts here.